react/modules/gaEvents.ts

```diff
@@ -260,6 +261,23 @@ export function beginCheckout(eventData: BeginCheckoutData) {
   updateEcommerce(eventName, { ecommerce: data })
 }
 
+export function addShippingInfo(eventData: AddShippingInfoData) {
+  const eventName = 'add_shipping_info'
+
+  const { currency, items: eventDataItems, shippingTier, value } = eventData
+
+  const { items } = formatCartItemsAndValue(eventDataItems ?? [])
```

Comment:
I think we can skip this check now that `item` property type isn't optional anymore.
```suggestion
  const { items } = formatCartItemsAndValue(eventDataItems)
```